ui/components/app/srp-input/srp-input.js

```diff
@@ -0,0 +1,112 @@
+import { ethers } from 'ethers';
```

Comment:
This looks to be a clean refactor and extraction of the code that was in ui/components/app/create-new-vault/create-new-vault.js